sonar-csharp-plugin/pom.xml

```diff
@@ -4,7 +4,7 @@
 
   <parent>
     <groupId>org.sonarsource.dotnet</groupId>
-    <artifactId>sonar-csharp</artifactId>
+    <artifactId>sonar-dotnet</artifactId>
```

Comment:
You've changed the artifactId which is good but not the folder which make it inconsistent

Reply:
This kind of confusion is exactly the motivation for the fix 😃 There are two things:

`sonar-csharp` - old root, renamed to `sonar-dotnet`
`sonar-csharp-plugin` - the real plugin, submodule and directory that remained unchanged. Only it's `<parent>` was updated to the new one.

Reply:
Indeed didn't check that it was the parent tag modified here, so let's say I did not write this comment ;)